crates/cxx-qt-gen/src/parser/cxxqtdata.rs

```diff
@@ -76,6 +90,8 @@ impl ParsedCxxQtData {
                 };
 
             // Read each of the types in the mod (type A;)
+            //
+            // TODO: need to read the functions
```

Comment:
add in separate commit

Reply:
We don't write out functions from the extern blocks, so we don't need to do this ? The inherit branch #363 might need to consider some of this :thinking: